TODO.md

```
@@ -30,12 +30,7 @@ These items have dependencies and should be done in order.
 
 ## To plan/design:
 
-### Configuration system (should come first - unblocks big lifts)
-- Need to develop a configuration system so everything can be configurable. Need heavy inspiration from opencode. E.g. similar "here is the schema" header stuff as their config and similar granularity of configurable options available.
-- This absolutely needs to come before any "big lifts" or "large changes" (e.g. before database or before doing more agent work).
-- Will provide proper place to specify: thoughts root, clones dir, DB path, profiles, etc.
-
-### SQLite migration for thoughts (blocked by: config system)
+### SQLite migration for thoughts (blocked by: config system, which is mostly done now)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)